spackbot/routes.py

```diff
@@ -125,3 +130,10 @@ async def label_pull_requests(event, gh, *args, session, **kwargs):
     Add labels to PRs based on which files were modified.
     """
     await handlers.add_labels(event, gh)
+
+@router.register("pull_request", action="closed")
+async def on_closed_pull_request(event, gh, *args, session, **kwargs):
+    """
+    Respond to the pull request closed
+    """
+    await handlers.graduate_pr_binaries(event, gh)
```

Comment:
I think `handlers/__init__.py` still needs to import this to make it available as you've invoked it here.

Reply:
so i.e.:

```
from .mirrors import graduate_pr_binaries
```